src/scripts.js

```diff
@@ -48,78 +76,169 @@ function uploadData() {
   currentUser = new User(userData[getRandomIndex(userData)])
   currentUserTeam = currentUser.friends.map(friend => userData[friend-1])
   currentUserTeam.unshift(currentUser) // NOTE: currentUser will always be at [0]
-  // WE CAN CHANGE THIS IF WE NEED TO ^
   currentUserRepository = new UserRepository(currentUserTeam)
```

Comment:
John Adams was saying when we insantiate a new user, we should hook up all the other classes as parameters? Maybe something to explore a little more....IF WE HAD TIME.